### ISSUE TITLE
New login panel design

### ISSUE THREAD

**Issue by saml1er** [drunkdriversclub/ddc#4]
**Is your feature request related to a problem? Please describe.**
The current login panel doesn't look good at all. It's very basic. Almost all race servers on MTA have good design. 

**Describe the solution you'd like**
A new login panel with nice design. We'll have to completely avoid using CEGUI. The login panel should have smooth animations.

**Comment by ZUNII** [comment]
I already saw a prototype of Timbo's design and it looks damn good I hope we get his design

**Comment by saml1er** [comment]
> 
> 
> I already saw a prototype of Timbo's design and it looks damn good I hope we get his design

Yes, I'm working with tim. Sando also joined recently. He's helping with design ideas. The login panel is almost done, and I'm very close to releasing it.
